modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -71,107 +53,91 @@ You need to select two different sets of channels, one for {es} and the other fo
 You will need an activation key associated with the correct {es} channels.
 For more information about activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[].
 
+ifeval::[{suma-content} == true]
 
-[IMPORTANT]
-====
-For {es} 8 clients, add both the ``BaseOS`` and ``Appstream`` channels.
-You will require packages from both channels.
-If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
-====
+The channels you need for this procedure are:
 
+[[es-channels-wizard]]
+[cols="1,1,1", options="header"]
+.ES Channels - WebUI
+|===
+| OS Version | Base Channel                     | Tools Channel
+| {es} 6     | RHEL Expanded Support 6          | RES6 SUSE-Manager-Tools x86_64
+| {es} 7     | RHEL Expanded Support 7          | RES7 SUSE-Manager-Tools x86_64
+| {es} 8     | RHEL or SLES ES or CentOS 8 Base | RHEL8-Pool for x86_64
+|===
 
-.Procedure: Adding Client Tools Channels
+include::snippets/add_channels_wizard.adoc[]
 
-. On the {productname} Server, add the appropriate {es} channels:
-+
-* For {es} 6:
-+
-From the {webui}, add [systemitem]``RHEL Expanded Support 6 x86_64``.
-+
-From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemitem]``res6-suse-manager-tools-x86_64``.
-+
-* For {es} 7:
-+
-From the {webui}, add [systemitem]``RHEL Expanded Support 7 x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES7 x86_64``.
-+
-From the command prompt, add [systemitem]``rhel-x86_64-server-7`` and [systemitem]``res7-suse-manager-tools-x86_64``.
-+
-* For {es} 8:
-+
-From the {webui}, add [systemitem]``RHEL8 Base x86_64`` and [systemitem]``SUSE Manager Tools for RHEL and ES 8 x86_64``.
```

Comment:
As said in my latest bug report, I think `RHEL8 Base x86_64` does not exist anymore.

But maybe you're planning to do that in a separate PR. If so, please ignore this remark.